src/lib/virtualendpoint/message_handler.ts

```diff
@@ -0,0 +1,40 @@
+import { Driver } from "zwave-js";
+import { UnknownCommandError, VirtualEndpointNotFoundError } from "../error";
+import { VirtualEndpointCommand } from "./command";
+import { IncomingMessageVirtualEndpoint } from "./incoming_message";
+import { VirtualEndpointResultTypes } from "./outgoing_message";
+
+export class VirtualEndpointMessageHandler {
+  static async handle(
+    message: IncomingMessageVirtualEndpoint,
+    driver: Driver
+  ): Promise<VirtualEndpointResultTypes[VirtualEndpointCommand]> {
+    const { command } = message;
+
+    const virtualNode =
+      "nodeIDs" in message
+        ? driver.controller.getMulticastGroup(message.nodeIDs)
+        : driver.controller.getBroadcastNode();
```

Comment:
We're creating a new 'virtualendpoint' namespace while ZJS doesn't do this. ZJS just uses `controller` for this, and so should we. 

In a perfect world a user should be able to figure out our commands by reading the ZJS (not server) API reference.

Reply:
Fair enough. What would you call the set_value command then? controller.set_value_multicast and _broadcast seem reasonable?

Reply:
Looking at it again, I guess you are fetching a node, so it could be a new endpoint. 

Does make me wonder if we should have instead `multicast_group` and `broadcast_node` be the namespaces for our commands.